readme.md

```diff
@@ -1,26 +1,36 @@
 ##GitFun
 
-GitFun is an interactive git workshop following the style of [githug](https://github.com/Gazler/githug). gitfun is geared towards understanding the git workflow in a team, and how to deal with common problems and errors.
+GitFun is an interactive git workshop following the style of [githug](https://github.com/Gazler/githug). GitFun is geared towards understanding the git workflow in a team, and how to deal with common problems and errors.
 
-### To run
+### Setup
 
-Note: This workflow is subject to change as the project grows
+Note: This workflow is subject to change as the project grows.
 
-```
-$ git clone https://github.com/kmacpher/gitfun
-$ cd gitfun
-$ npm link or $ npm install
-Then from any directory, you can run gitfun as a command line tool
-$ gitfun
+```sh
+git clone https://github.com/kmacpher/gitfun
+cd gitfun
+npm link
+# Now from any directory, you can run gitfun as a command line tool:
+gitfun
+cd gitfun_workshop
 ```
 
+*NOTE: if you get an error `cannot find OpenSSL or Libgcrypt`, run `xcode-select --install` and then try `npm link` again.*
```

Comment:
`nodegit` uses the Xcode developer tools to do some OpenSSL stuff. If the user's Xcode tools are missing or outdated — as frequently happens during OS updates — it's hard to debug the resulting error message. Hopefully this message staves off any concerned emails from Foundations students.